src/rev_ai/customvocabulariesclient.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using Rev.ai"""
+
+import requests
+import json
+from requests.exceptions import HTTPError
+# from .models import Job, Account, Transcript, CaptionType
+from . import __version__
+
+try:
+    from urllib.parse import urljoin
+except:
+    from urlparse import urljoin
+
+
+class RevAiCustomVocabulariesClient:
+    """Client which implements Rev.ai CustomVocabulary API"""
+
+    # Default version of Rev.ai
+    version = 'v1'
+
+    # Default address of the API
+    base_url = 'https://api.rev.ai/speechtotext/{}/vocabularies'.format(
+        version)
+
+    def __init__(self, access_token):
+        """Constructor
+
+        :param access_token: access token which authorizes all requests and
+                             links them to your account. Generated on the
+                             settings page of your account dashboard
+                             on Rev.ai
+        """
+        if not access_token:
+            raise ValueError('access_token must be provided')
+
+        self.default_headers = {
+            'Authorization': 'Bearer {}'.format(access_token),
+            'User-Agent': 'RevAi-PythonSDK/{}'.format(__version__)
+        }
```

Comment:
These are so similar to the apiclient.py class definition. Why do you choose to define the below functions in a new file instead of apiclient.py?

Reply:
I think it's a good idea to have it as a separate client (especially since these are not usable in the normal api currently, having them in the same client will lead to confusion)

I would do something similar to what we did in the node SDK and create a base class which both apiclient and customvocab client can extend. We do something similar in the node SDK